website/cue/reference/data_model/schema.cue

```diff
@@ -349,4 +349,41 @@ data_model: schema: {
 			}
 		}
 	}
+
+	trace: {
```

Comment:
I might suggest we leave out this documentation until we have a source / sink that uses traces otherwise users may be confused about why no components generate or support sending traces.

Reply:
That's probably a safe option, I will save that for later.